docs/develop-a-squid/squid-processor/evm-support.md

```diff
@@ -149,4 +149,4 @@ async function handleEvmLog(ctx: BatchContext<Store, unknown>, block: SubstrateB
 }
 ```
 
-Since the handlers are lazy (meaning that the database transaction is not opened if `ctx.store` is not hit), the performance overhead is minimal.
\ No newline at end of file
+Since the handlers are lazy (meaning that the database transaction is not opened if `ctx.store` is not hit), the performance overhead is minimal.
```

Comment:
That's a lie. This has a significant impact on performance.

Reply:
Okay let's remove it then